class.jetpack-cli.php

```diff
@@ -1884,6 +1884,7 @@ public function block( $args, $assoc_args ) {
 					'slug'            => $slug,
 					'title'           => $title,
 					'underscoredSlug' => str_replace( '-', '_', $slug ),
+					'jetpackVersion'  => substr( JETPACK__VERSION, 0, strpos( JETPACK__VERSION, '.' ) ) . '.x',
```

Comment:
Is this too much magic in one line? It's a little tricky to read immediately, but it's fairly simple logic.

Reply:
This seems fine. 👍